app/assets/tailwind/application.css

```
@@ -1,36 +1,267 @@
+    @apply inline-flex min-h-11 min-w-11 items-center justify-center rounded-full text-gray-600 transition;
+  }
+
+  .footer-icon-link:hover,
+  .footer-icon-link:focus-visible {
+    background: var(--brand-soft);
+    color: var(--brand-dark);
+    transform: translateY(-2px);
+  }
+
+  .footer-icon-link svg {
+    fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)